WordPress/Classes/ViewRelated/Stats/SiteStatsTableViewCells.swift

```diff
@@ -336,7 +336,7 @@ struct TotalInsightStatsRow: ImmuTableRow {
             return
         }
 
-        cell.configure(count: dataRow.count, difference: dataRow.difference, percentage: dataRow.percentage, sparklineData: dataRow.sparklineData, guideText: dataRow.guideText, guideURL: dataRow.guideURL, statSection: statSection, siteStatsInsightsDelegate: siteStatsInsightsDelegate)
+        cell.configure(dataRow: dataRow, guideURL: dataRow.guideURL, statSection: statSection, siteStatsInsightsDelegate: siteStatsInsightsDelegate)
```

Comment:
It seems to me like the 2nd parameter here is not necessary since you are passing dataRow in the first parameter.  I removed the 2nd parameter myself and only used the dataRow object and it worked fine.

Reply:
Goood observation 👍.  I somehow managed to miss it. 

<img src="https://media3.giphy.com/media/xT5LMzIK1AdZJ4cYW4/giphy.gif?cid=790b7611fc9592f9ba46eb2d3de0ea92c856f4d126340666&rid=giphy.gif&ct=g" alt="drawing" width="200"/>